asana/client.py

```diff
@@ -83,8 +83,16 @@ def request(self, method, path, **options):
                 response = getattr(self.session, method)(
                     url, auth=self.auth, **request_options)
                 if response.status_code in STATUS_MAP:
-                    raise STATUS_MAP[response.status_code](response)
-                elif response.status_code >= 500 and response.status_code < 600:
+                    # TODO: Change back to default behavior once 1.0 & 1.1 have
+                    # the same premium only response
+                    asana_error = STATUS_MAP[response.status_code](response)
+                    premium_only_str = "not available for free"
```

Comment:
Constants in Python should be defined as `UPPER_SNAKE_CASE` and also be defined as class- or module-level variables rather than being instantiated with every call. I would move this to right below `ALL_OPTIONS`, and would also prefix it with an underscore to signify that this only has meaning for the internal implementation: `_PREMIUM_ONLY_STR`.

Reply:
Makes sense, +1.